flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/catalog/CatalogManager.java

```diff
@@ -327,11 +328,15 @@ public void createCatalog(String catalogName, CatalogDescriptor catalogDescripto
      *
      * @param catalogName the given catalog name under which to alter the given catalog
      * @param catalogUpdater catalog configuration updater to alter catalog
+     * @param catalogCommentUpdater catalog comment updater to alter catalog
      * @throws CatalogException If the catalog neither exists in the catalog store nor in the
      *     initialized catalogs, or if an error occurs while creating the catalog or storing the
      *     {@link CatalogDescriptor}
      */
-    public void alterCatalog(String catalogName, Consumer<Configuration> catalogUpdater)
+    public void alterCatalog(
+            String catalogName,
+            Consumer<Configuration> catalogUpdater,
+            Function<String, String> catalogCommentUpdater)
```

Comment:
That's not a good idea. Maybe we should support something like `CatalogChange` to update the `Catalog`, similar to how `TableChange` works.

Reply:
I've created a [PR](https://github.com/liyubin117/flink/pull/2) to refactor this method.